Root/OptimizationDump.cxx

```diff
@@ -377,13 +377,17 @@ EL::StatusCode OptimizationDump :: execute ()
   }
 
   if(!m_inputMET.empty() && !m_inputJets.empty()){
-    m_effectiveMass = VD::Meff_inclusive(in_met, in_jets, in_muons, in_electrons)/1000.;
+    
+    ConstDataVector<xAOD::JetContainer> presel_jets = VD::subset_using_decor(in_jets, VD::decor_vd_pass_Presel, 1);
+    //std::cout << in_jets->size() << " " << presel_jets.size() << std::endl;
+    m_effectiveMass = VD::Meff_inclusive(in_met, presel_jets.asDataVector(), in_muons, in_electrons)/1000.;
```

Comment:
It wouldn’t, would it? The definition of the jet object is in SR1, as everywhere, pT > 30, |eta| < 2.8, passes OR, etc. We never change this in our various regions, and have no plans to.

On Oct 15, 2015, at 8:58 PM, Giordon Stark <notifications@github.com<mailto:notifications@github.com>> wrote:


In Root/OptimizationDump.cxx<https://github.com/kratsg/TheAccountant/pull/136#discussion_r42202484>:

> @@ -377,13 +377,17 @@ EL::StatusCode OptimizationDump :: execute ()
>    }
>
>    if(!m_inputMET.empty() && !m_inputJets.empty()){
> -    m_effectiveMass = VD::Meff_inclusive(in_met, in_jets, in_muons, in_electrons)/1000.;
> +
> +    ConstDataVector<xAOD::JetContainer> presel_jets = VD::subset_using_decor(in_jets, VD::decor_vd_pass_Presel, 1);
> +    //std::cout << in_jets->size() << " " << presel_jets.size() << std::endl;
> +    m_effectiveMass = VD::Meff_inclusive(in_met, presel_jets.asDataVector(), in_muons, in_electrons)/1000.;


Thanks! This still perplexes me. What if the preselection becomes.. for example... the signal region definition, SR-1? Then calculations of Meff could potentially change!

—
Reply to this email directly or view it on GitHub<https://github.com/kratsg/TheAccountant/pull/136/files#r42202484>.